imageroot/actions/get-configuration/10get

```diff
@@ -16,14 +16,13 @@ import subprocess
 
 cloud_log_manager = {}
 
-with subprocess.Popen(['systemctl', '--user', 'is-active', 'cloud-log-manager-forwarder'], stdout=subprocess.PIPE, stderr=sys.stderr) as pipe:
-    response = pipe.stdout.readline().decode('utf-8').replace('\n', '')
-    if 'inactive' in response:
-        cloud_log_manager["status"] = 'inactive'
-    elif 'active' in response:
-        cloud_log_manager["status"] = 'active'
-    elif 'failed' in response:
-        cloud_log_manager["status"] = 'failed'
+response = subprocess.run(['systemctl', '--user', 'is-active', 'cloud-log-manager-forwarder'], capture_output=True, text=True)
+if 'inactive' in response.stdout:
+    cloud_log_manager["status"] = 'inactive'
+elif 'active' in response.stdout:
+    cloud_log_manager["status"] = 'active'
+elif 'failed' in response.stdout:
+    cloud_log_manager["status"] = 'failed'
```

Comment:
This can be a [match](https://peps.python.org/pep-0636/) if you want

Reply:
I used it in a past commit but i think i lost these changes in some git mistakes :laughing:

Reply:
:hocho: